docs/reference/ml/df-analytics/apis/put-trained-model-definition-part.asciidoc

```diff
@@ -0,0 +1,56 @@
+[role="xpack"]
+[testenv="basic"]
+[[put-trained-model-definition-part]]
+= Put trained model definition part API
+[subs="attributes"]
+++++
+<titleabbrev>Create trained models</titleabbrev>
+++++
+
+Puts part of a trained model definition.
+experimental::[]
+
+[[ml-put-trained-model-definition-part-request]]
+== {api-request-title}
+
+`PUT _ml/trained_models/<model_id>/definition/<part_num>`
+
+
+[[ml-put-trained-model-definition-part-prereq]]
+== {api-prereq-title}
+
+Requires the `manage_ml` cluster privilege. This privilege is included in the
+`machine_learning_admin` built-in role.
+
+
+[[ml-put-trained-model-definition-part-path-params]]
+== {api-path-parms-title}
+
+`<model_id>`::
+(Required, string)
+include::{es-repo-dir}/ml/ml-shared.asciidoc[tag=model-id]
+
+`<part_num>`::
+(Required, number)
+The definition part number. When the definition is loaded for inference
+the definition parts will be streamed in order of their `part_num`.
```

Comment:
This should state that `part_num` is 0 based indexing.

"The first part must have `part_num` equal to 0 and the final part equal to `total_parts -1`